L1Trigger/L1TCalorimeter/src/firmware/Stage2Layer2DemuxSumsAlgoFirmwareImp1.cc

```diff
@@ -144,6 +144,14 @@ void l1t::Stage2Layer2DemuxSumsAlgoFirmwareImp1::processEvent(const std::vector<
       }
   }
 
+  // calculate centrality
```

Comment:
@rekovic it looks like this file revert the changes in #24752, could you please clarify?

Reply:
@fabiocos 
The code is reorganized for clarity, and further fixes for saturation are implemented.
Attn @bundocka

Reply:
this should now be correct, it took a few attempts to get the saturation handling in the emulator to match firmware due to some subtleties.